gsa/src/web/pages/policies/component.js

```diff
@@ -751,12 +753,13 @@ class PolicyComponent extends React.Component {
         )}
         {editPolicyFamilyDialogVisible && (
           <EditPolicyFamilyDialog
-            policy={policy}
-            policyName={policyName}
+            config={policy}
+            config_name={policyName}
             family_name={family_name}
             id={id}
             nvts={nvts}
             selected={selected}
+            text={_('Policy')}
```

Comment:
Could you rename this to something more meaningful from which it is possible to figure out what the text is used for?